google-cloud-spanner/src/main/java/com/google/cloud/spanner/Statement.java

```diff
@@ -86,6 +86,12 @@ private Builder(Statement statement) {
           statement.queryOptions == null ? null : statement.queryOptions.toBuilder().build();
     }
 
+    /** Replaces the current SQL of this builder with the given string. */
+    public Builder withSql(String sql) {
```

Comment:
Nit - Should we name this `replace` or `replaceWithSql` ?

Reply:
I chose the `withSql` name to be consistent with the existing `withQueryOptions` method on this class, which also replaces any existing value. I'm happy to change it to something else if you feel strongly about this.

Reply:
In this case the naming convention is more like existing `append()` method. So going by that we can name this `replace()`.